src/dmd/astenums.d

```diff
@@ -0,0 +1,354 @@
+/**
+ * Defines enums common to dmd and dmd as parse library.
+ *
+ * Copyright:   Copyright (C) 1999-2021 by The D Language Foundation, All Rights Reserved
```

Comment:
These years seem to be quite arbitrary throughout the front-end. Can we just only have the current year for new sources? The update-copyright script already caters for detecting and rewriting `2021` into `2021-2022`.

Reply:
Since the code is copied from other files, the copyright notice should be the broadest of the files it was copied from.